src/infinity/libraries/LibRetire.sol

```diff
@@ -28,7 +29,8 @@ library LibRetire {
         MOSS,
         C3,
         ICR,
-        COOREST
+        COOREST,
+        CMARK
```

Comment:
Just a note to update the subgraphs before doing a production retirement, otherwise they will break from the unknown registry not found in the enum.

This PR: https://github.com/KlimaDAO/klima-subgraph/pull/178 would fix that error. However I've held off on merging this as I wanted to investigate whether the `unsupported_registry` would actually bubble up and break the api.

Reply:
did that here in a standalone PR: https://github.com/KlimaDAO/klima-subgraph/pull/329